src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -127,4 +127,3 @@ private static AnalyzeEntitiesResponse analyzeEntity(String message) throws IOEx
     }
   }  
 }
-
```

Comment:
[Nit] it seems like you deleted a newline at the end of file here, Github's making it hard to tell.

Reply:
There were two trailing new lines